cosmwasm/enclaves/shared/contract-engine/src/db.rs

```diff
@@ -11,7 +11,7 @@ use crate::external::{ecalls, ocalls};
 use super::contract_validation::ContractKey;
 use super::errors::WasmEngineError;
 
-#[cfg(not(feature = "query-only"))]
+// #[cfg(not(feature = "query-only"))]
```

Comment:
Why all of these?

Reply:
To answer both your comments:

With this new design we don't need two enclaves because it supports both the query-only and regular workflows. For now I'm just commenting things out but imo we can just remove the query enclave completely